src/lib/builtins.pl

```diff
@@ -33,6 +33,137 @@
 internal settings and basic I/O are all here.
 */
 
+%% (@>)(?X, ?Y)
```

Comment:
This is not valid syntax, I have filed https://github.com/mthom/scryer-prolog/issues/2270 for this.

Reply:
It is valid to write it as `@>(?X, ?Y)` (with suitable prefix operator `?`).